### PR TITLE
EMP + BoH = emptied BoH, messy floor.

### LINKED ISSUE #8635: Bag of holding nerf and balance suggestion
When EMPed, make it so the bag of holding malfunctions and disappears, such as when a bagulo happens. Look at all those gadgets on the BoH, how could it not malfunction when EMPed? I am not suggesting a singulo happen over a single bag, but I am suggesting that the bag itself just disappears out of your hands/back and all items inside are deleted when it is sucked into the malfunction.

Also don't be a retard and make sure there's no oversight for important items such as the nuke disk.

As it stands, the bag of holding is byfar the best bag in the game and when produced, everyone has one like it's a fashion statement. However, with this nerf we can suggest some balance to it too.

Maybe you can print a new stackable item to put inside your bag that allows EMP shielding, such as the EMP shielding pAIs get and security cameras get when plasma is involved, and this would imply the EMP shielding would be related to plasma. When EMPed, the EMP shield will consume stacks depending on the strength of the EMP and your bag would stay safe. Hell, this EMP protection doesn't even have to take a backpack slot, maybe you can attach it to part of the backpack a separate way so it doesn't take inventory space, and when examined the bag will tell you how much EMP protection it has.

### PR DESCRIPTION
**Pull request by Intigracy**

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -90,6 +90,16 @@
 	qdel(src)
 	return
 
+/obj/item/weapon/storage/backpack/holding/emp_act(severity)
+	if(prob(50+severity))
+		for(var/obj/O in src.contents)
+			if(istype(src.loc, /mob))
+				var/mob/user = src.loc
+				O.loc = user.loc
+				user.visible_message("<span class = 'danger'>The Bluespace portal malfunctions, spitting out its contents onto the floor!</span>")
```
Comment (9600bauds) [inline code/game/objects/items/weapons/storage/backpack.dm]: Are you sure you want this message once for every item

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -90,6 +90,16 @@
 	qdel(src)
 	return
 
+/obj/item/weapon/storage/backpack/holding/emp_act(severity)
+	if(prob(50+severity))
+		for(var/obj/O in src.contents)
+			if(istype(src.loc, /mob))
+				var/mob/user = src.loc
+				O.loc = user.loc
```
Comment (9600bauds) [inline code/game/objects/items/weapons/storage/backpack.dm]: ForceMove

**Comment by 9600bauds** [comment]
It helps to say what the pr does in the description so that people don't have to hunt down the changelog

**Comment by ihadtoregisterforthis** [comment]
What is this, request fixing week?

**Comment by 9600bauds** [comment]
It's to autoclose the issue you nerd

**Comment by FudgePucker** [comment]
Oh so you implemented a controversial feature, and did not describe in your PR what the fuck you did?
Great job
:joy: :joy: :joy: :ok_hand: :ok_hand: :ok_hand:

**Comment by Exxion** [comment]
You know, you don't have to say "fixes"
"Closes" and "addresses" also work

**Comment by Intigracy** [comment]
I don't care about what word I use to close the issue. I'm going to keep using fixes # because it makes everyone's autism flare up.

>did not describe PR what I did

**Comment by Intigracy** [comment]
Thank god nobody who has collaborator actually merges things without looking at the code/changelog portion of it, right @FudgePacker? But since you're too lazy to do that, I updated the OP with language you can understand.

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 2 above)
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/backpack.dm]: thanks fam, forgot

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 1 above)
Comment (9600bauds) [inline code/game/objects/items/weapons/storage/backpack.dm]: Didn't that come up during testing anyways?

Comment (Intigracy) [inline code/game/objects/items/weapons/storage/backpack.dm]: Yeah, I had like 6 BoH's on the ground though so figured it was just all of them dumping contents.

**Comment by 9600bauds** [comment]
dreamseeker_2016-03-15_11-27-51
Intigracy my man I believe in the time it took you to write that passive-aggressive OP you could have actually tested this fifteen times over because it didn't take me ten seconds to find out how horribly broken it was
You lose your tested seal privileges

**Pull request closed without merging**